static/app/views/replays/replays.tsx

```diff
@@ -34,34 +36,59 @@ type Props = AsyncView['props'] &
     statsPeriod?: string | undefined; // revisit i'm sure i'm doing statsperiod wrong
   };
 
-class Replays extends React.Component<Props> {
-  getEventView() {
-    const {location, selection} = this.props;
-    const {query} = location;
+const sanitizeLocationQuery = query => {
```

Comment:
`sanitize` isn't totally accurate, we're making sure it's a string and ignoring other possible data types. if we had a query string `?query=foo&query=bar`, `location.query.query` in this case would be `['foo', 'bar']`. This shouldn't happen in practice, but now that I'm typing this out, we could support this by joining the array with a space.

maybe `getQueryParamAsString`?

Would also be good to add a comment for this fn.

Reply:
I'll add the join and update the function name and add a comment explaining the intent better.